src/modules/addRemove.js

```diff
@@ -0,0 +1,174 @@
+import Task from './task.js';
+
+class displayTask {
+  static displayTasks() {
+    const tasks = Store.getTasks();
+    tasks.forEach((task) => displayTask.addTaskToList(task));
+  }
+
+  static addTaskToList(task) {
+    const elementList = document.querySelector('.list');
+    const listItem = document.createElement('li');
+    listItem.setAttribute('class', 'item');
+
+    listItem.innerHTML = `<input type="checkbox" id='${task.id}' name="taskStatus" class = 'taskStatus'>
+                    <p class='description edit' contenteditable='true' id='${task.id}'>${task.description}</p>
+                    <i class="fas fa-trash-alt remove-btn" id='${task.id}'></i>`;
+
+    elementList.appendChild(listItem);
+  }
+
+  static deleteTask(el) {
+    if (el.classList.contains('remove-btn')) {
+      el.parentElement.remove();
+    }
+  }
+
+  static clearFields() {
+    document.querySelector('#inputTask').value = '';
+  }
+}
+
+class Store {
+  static getTasks() {
+    let tasks;
+    if (localStorage.getItem('tasks') === null) {
+      tasks = [];
+    } else {
+      tasks = JSON.parse(localStorage.getItem('tasks'));
+    }
+
+    return tasks;
+  }
+
+  static addTask(task) {
+    const tasks = Store.getTasks();
+    tasks.push(task);
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  }
+
+  static removeTask(elem) {
+    if (elem.classList.contains('remove-btn')) {
+      const tasks = Store.getTasks();
+      if (tasks.length === 1) {
+        tasks.pop();
+      } else {
+        tasks.splice(parseInt(elem.id), 1);
+
+        tasks.forEach((element, index) => {
+          element.id = index + 1;
+        });
+      }
+      index = tasks.length;
+
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    }
+  }
+
+  static modifyTask(elem) {
+    const tasks = Store.getTasks();
+    console.log(elem.id);
```

Comment:
- [ ] It is not good practice to push console logs. They kill performance and expose confidential data in more sophisticated apps. Kindly remove it.